db/connect.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const connectDB = async () => {
+  const url = 'mongodb+srv://jukaririchardson:TwentyTwenty3*@cluster0.yjrtwue.mongodb.net/';
+
+  try {
+    await mongoose.connect(url, {
+      useNewUrlParser: true,
+    //   useCreateIndex: true,
+    //   useFindAndModify: false,
+      useUnifiedTopology: true,
+    });
+    console.log('Connected to MongoDB');
+  } catch (error) {
+    console.error('Error connecting to MongoDB:', error);
+    throw error;  // Propagate the error so that the calling code can handle it
+  }
+};
+
```

Comment:
You should remove the db folder that I created earlier as you are using this one. Also remove the auth.js file from controllers folder that I created, as we don't need it anymore.